packages/server/src/routes/grants.js

```diff
@@ -235,8 +235,15 @@ router.delete('/:grantId/interested/:agencyId', requireUser, async (req, res) =>
     const { user } = req.session;
     const { grantId, agencyId } = req.params;
     const { agencyIds } = req.body;
-    console.log(agencyIds);
-    const allowed = await isUserAuthorized(user, agencyIds || agencyId);
+    let authorizedAgencyId = agencyId;
+    if (agencyIds.length > 0) {
+        // Note: ran into a ES lint error so had to declare the variable first
+        // and then assign it to authorizedAgencyId.
+        const [firstAgency] = agencyIds;
+        authorizedAgencyId = firstAgency;
+    }
+    const allowed = await isUserAuthorized(user, authorizedAgencyId);
```

Comment:
you could do something like this:
```
const allowed = _.every(await Promise.all(
  agencyIds.map(agencyId => isUserAuthorized(user, agencyId))
));
```
...but the query that `inTenant` does has some joins, it's not amazing to do n of them concurrently.

Better yet, I'd just create a version of the helper like `isUserAuthorizedForMultipleAgencies`; should be easy since `isUserAuthorized` just calls `inTenant` right now, which already supports multiple agencies.

Reply:
I agree with your hesitation to run extra `inTenant` queries concurrently. 

Rather than introduce a new helper function, I made an update that keeps the existing `isUserAuthorized` compatible with existing calls by making the second parameter of that function variadic. That way, it's still possible to only pass a single agency ID value as the second parameter (as any extant calls will be doing), but also allow multiple agency IDs to be passed in for cases like this one.

This way, if more than zero agency IDs are passed in the body, we will use (all of, and only) those values for both the permission check and the subsequent `DELETE` query, and the `agencyId` request param is completely ignored. Otherwise, if request body's `agencyIds` value is empty, we use the singular `agencyId` request param value as a "fallback" and only evaluate that ID.